meta-balena-common/recipes-kernel/linux/kernel-devsrc.bb

```diff
@@ -0,0 +1,256 @@
+SUMMARY = "Linux kernel Development Source"
```

Comment:
It's not clear to me what we fetched from upstream here. Did we fetch the newer recipe that creates the sanitized headers archive or we fetched the recipe that packages the full source? Can this info be maybe part of the commit log?

Reply:
^ ?

Reply:
@floion 
I did add a commit message. https://github.com/balena-os/meta-balena/pull/1605/commits/c7e5c28db2ea500c25ed75e311e104fbebfee2cb

```
Since thud, kernel-devsrc changed from tarballing the full kernel
source to just the headers. Lets include the recipe from upstream so
that we have a consistent set of kernel-headers-source generated
accross all devices. Otherwise, devices on older yocto versions will be
generating a full kernel source tarball while devices running new
yocto versions will be generating just the headers tarball.

Fetched this specific version
http://cgit.openembedded.org/openembedded-core/tree/meta/recipes-kernel/linux/kernel-devsrc.bb?id=68b9243729ac1982c9f1d30980cf979267bbc7c3

Signed-off-by: Zubair Lutfullah Kakakhel <zubair@balena.io>
```

Reply:
Please make this clear in the commit log.